### PR TITLE
CID-9 | ContextMenu

### PR DESCRIPTION
**Pull request by WallochMatt**
Feature to use a right-click context menu on items of a given page. Menu spawns only on click for a table item (row of a users, location or group).

### CODE REVIEW

**Comment by WallochMatt** [comment]
contextMenu
^On right click of a row item (user, location or group), this context menu spawns. For now, clicking an option only sends an alert.

File: Frontend/src/ContextMenu.svelte [hunk 1]
```diff
@@ -1,154 +1,67 @@
...
 
     let menuItems = [
         {
-            'name': 'addItem',
-            'onClick': addItem,
-            'displayText': "Add Item",
+            'name': 'editItem',
+            'onClick': editItem,
+            'displayText': "Edit",
             'icon': 'font awesome'
```
Comment (ChrisHaliga) [inline Frontend/src/ContextMenu.svelte]: I'm assuming 'font awesome' wasn't what you wanted for your icon

**Comment by ChrisHaliga** [comment]
Both PRs look good. Let me know if you have any trouble resolving any merge conflicts

File: Frontend/src/ContextMenu.svelte (hunk 1 above)
Comment (WallochMatt) [reply Frontend/src/ContextMenu.svelte]: It was more so a reminder to use the tool later.

Comment (ChrisHaliga) [reply Frontend/src/ContextMenu.svelte]: Gotcha. I would just rip it out and put it back later, should you need icons. Definitely wouldn't recommend having reminders in the data

Comment (WallochMatt) [reply Frontend/src/ContextMenu.svelte]: Good point. I did not realize how confusing it appeared. I'll delete them.

**Pull request merged**